go.mod

```diff
@@ -0,0 +1,41 @@
+module github.com/openshift/service-ca-operator
+
+go 1.12
+
+require (
+	github.com/certifi/gocertifi v0.0.0-20180905225744-ee1a9a0726d2 // indirect
+	github.com/coreos/bbolt v1.3.2 // indirect
+	github.com/coreos/go-systemd v0.0.0-20190321100706-95778dfbb74e // indirect
+	github.com/coreos/pkg v0.0.0-20180928190104-399ea9e2e55f // indirect
+	github.com/davecgh/go-spew v1.1.1
+	github.com/getsentry/raven-go v0.0.0-20190513200303-c977f96e1095 // indirect
+	github.com/golang/groupcache v0.0.0-20190129154638-5b532d6fd5ef // indirect
+	github.com/google/btree v1.0.0 // indirect
+	github.com/google/uuid v1.1.1
+	github.com/grpc-ecosystem/go-grpc-middleware v1.0.0 // indirect
+	github.com/grpc-ecosystem/grpc-gateway v1.9.0 // indirect
+	github.com/jteeuwen/go-bindata v3.0.8-0.20151023091102-a0ff2567cfb7+incompatible
+	github.com/openshift/api v0.0.0-20191119170142-082f8e2a947e
+	github.com/openshift/client-go v0.0.0-20191022152013-2823239d2298
+	github.com/openshift/library-go v0.0.0-20191118102510-4e2c7112d252
+	github.com/pkg/profile v0.0.0-20191120004908-e6812cd8b672 // indirect
+	github.com/prometheus/client_golang v0.9.3
+	github.com/soheilhy/cmux v0.1.4 // indirect
+	github.com/spf13/cobra v0.0.5
+	github.com/spf13/pflag v1.0.5
+	github.com/tmc/grpc-websocket-proxy v0.0.0-20190109142713-0ad062ec5ee5 // indirect
+	github.com/xiang90/probing v0.0.0-20190116061207-43a291ad63a2 // indirect
+	go.etcd.io/bbolt v1.3.2 // indirect
+	go.uber.org/atomic v1.4.0 // indirect
+	go.uber.org/multierr v1.1.0 // indirect
+	go.uber.org/zap v1.10.0 // indirect
+	k8s.io/api v0.0.0-20190918155943-95b840bb6a1f
+	k8s.io/apiextensions-apiserver v0.0.0-20190918161926-8f644eb6e783 // indirect
+	k8s.io/apimachinery v0.0.0-20190913080033-27d36303b655
+	k8s.io/client-go v0.0.0-20190918160344-1fbdaa4c8d90
+	k8s.io/component-base v0.0.0-20190918160511-547f6c5d7090
+	k8s.io/klog v0.4.0
+	k8s.io/kube-aggregator v0.0.0-20190918161219-8c8f079fddc3
```

Comment:
Use `replace` directives to "pin" the k8s versions.

The [goodmod tool](https://github.com/mfojtik/goodmod) can help with this.

Reply:
Is this documented somewhere?

I've updated quite a few repos using gomod before and it's generally sufficient to `go get` the appropriate revision of dependencies that need to be updated (e.g. `go get `k8s.io/kube-aggregator@<version>`).

In the case of the versions that are specified here, they are identical to the versions used by glide.

I'm sure there's good reason for using something like goodmod for repos where vendoring is complicated, but I'm loath to have to require it for simple repos if I already understand how to use go mod.

cc: @mfojtik